doc/man/scons.xml

```diff
@@ -1729,27 +1729,28 @@ Also suppresses SCons status messages.</para>
   </varlistentry>
 
   <varlistentry id="opt-site-dir">
-  <term><option>--site-dir=<replaceable>dir</replaceable></option></term>
+  <term><option>--site-dir=<replaceable>path</replaceable></option></term>
```

Comment:
It should be a dir right? it's not a path list, and folks may confuse it as such?